persistence/sql/persister_oauth2.go

```diff
@@ -301,9 +305,12 @@ func (p *Persister) InvalidateAuthorizeCodeSession(ctx context.Context, signatur
 	return sqlcon.HandleError(
 		p.Connection(ctx).
 			RawQuery(
```

Comment:
QueryWith

Reply:
I'm not sure if it's possible to combine RawQuery and QueryWith. Would you elaborate on this suggestion?